shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -1135,8 +1140,9 @@ public void updateSystemUiOverlays() {
   }
 
   private boolean stillAttachedForEvent(String event) {
-    if (delegate == null) {
-      Log.w(TAG, "FlutterActivity " + hashCode() + " " + event + " called after release.");
+    if (delegate == null || !delegate.isAttached()) {
+      Log.w(
+          TAG, "FlutterActivity " + hashCode() + " " + event + " called after release or detach.");
```

Comment:
can this avoid (either/or)?  If someone is looking at this log, it's always better to know exactly what is going on.
I'd suggest to break this log into two.

Reply:
done.